### DISCUSSION TITLE
What is AiClient interface

### DISCUSSION THREAD

**Discussion by mjosh1982** [spring-projects/spring-ai#565] [General]
In most of the discussions on the net, the examples given provides some AiClient interface. I do not find it anywhere when I use the open spring boot started project
import org.springframework.ai.client.AiClient;
Does anyone know how to get hold this interface and how to add specific dependency for the same.

**Comment by ThomasVitale** [comment]
@mjosh1982 `AiClient` was the early name of what later became `ChatClient`. The change was delivered last December and you can find more details about the renaming here: https://docs.spring.io/spring-ai/reference/upgrade-notes.html#_december_19_2023_update.

**Comment by B7-9414** [comment]
what I understand from @mjosh1982 question is .... Do we have a chatbot interface out-of-the-box something like Eureka client UI for example all we need doing the configuration or are going to have something like streamlit lib in java/spring AI . @ThomasVitale
